app/models/user.rb

```diff
@@ -1,7 +1,12 @@
 require 'securerandom'
 require 'audit'
+require 'open3'
 
 class User < ApplicationRecord
+  MAGIC = {png: ["\x89PNG\x0d\x0a\x1a\x0a"],
```

Comment:
Eventually, probably.  If it gets any more complicated or if it gets reused anywhere, sure.